CMakeLists.txt

```diff
@@ -482,6 +486,12 @@ add_custom_target(dpdk_includes ALL
 )
 endif ()
 
+if (ENABLE_P4TC)
```

Comment:
Shouldn't this be in the back end directory?

Reply:
Moved to backend